src/NexusMods.App/Services.cs

```diff
@@ -63,6 +64,7 @@ public static IServiceCollection AddApp(this IServiceCollection services,
                 .AddLibrary()
                 .AddLibraryModels()
                 .AddJobMonitor()
+                .AddLoadoutsSynchronizers()
```

Comment:
Is this still needed? I think it might not since you moved the models out of the Loadout.Synchronizer project

Reply:
```suggestion
```